.github/workflows/publish-vscode-extension.yaml

```diff
@@ -17,8 +17,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Build project
-        uses: ./.github/actions/build-project
+      # todo verify caching works well here too
+      - name: Setup and build project
+        uses: ./.github/actions/setup-and-build
```

Comment:
not sure how to best check for this

Reply:
We could just make a card and check the next time we do a VSCode release?